curriculum/challenges/english/14-responsive-web-design-22/learn-html-forms-by-building-a-registration-form/60fab8367d35de04e5cb7929.md

```diff
@@ -1,8 +1,8 @@
 ---
 id: 60fab8367d35de04e5cb7929
-title: Step 27
```

Comment:
We are going from step 27 to 29 is this correct?

Reply:
do you mean as the change? there is also a new step 17 so yes

Reply:
the name file changes were all automatic